tweets/2021-07/testgrid-into.tweet

```diff
@@ -0,0 +1 @@
+Introduction to the AI supported continuous integration project. Watch here! https://youtu.be/lY75bDv6kd4  This video details the TestGrid ecosystem and data sources as well as existing tools to access and prepare the data for analysis with python.
```

Comment:
Should we refer to TestGrid as testgrid.k8s.io? Because I see that there some other companies called TestGrid.

Reply:
:+1:

Reply:
thanks @oindrillac also made the edit on the other testgrid tweet